data/insights-for-openshift.yml

```diff
@@ -26,3 +26,6 @@ insights for openshift:
     OCP Advisor - Clusters - Details:
       url_rules:
         - /openshift/insights/advisor/clusters/**
+    Vulnerability - All Pages:
```

Comment:
Should we change this to 
```suggestion
    OCP Vulnerability - All Pages:
```
To keep it the same as advisor?

Reply:
Oh yes probably